perf/ssl_ctx.c

```diff
@@ -49,23 +48,24 @@ static void do_create_ctx(size_t num)
             goto out;
         if (mode == OP_SERVER) {
             if ((SSL_CTX_use_certificate_file(ctx, certpath,
-                                              SSL_FILETYPE_PEM) == 1) ||
+                                              SSL_FILETYPE_PEM) != 1) ||
                 (SSL_CTX_use_PrivateKey_file(ctx, keypath,
-                                             SSL_FILETYPE_PEM) == 1))
+                                             SSL_FILETYPE_PEM) != 1)) {
+                err = 1;
```

Comment:
There is also a `goto out` in the `if (ctx == NULL)` check above. Shouldn't we set `err = 1` there too?